makefiles/stdio.inc.mk

```diff
@@ -10,6 +10,7 @@ STDIO_MODULES = \
   stdio_uart \
   stdio_telnet \
   stdio_tinyusb_cdc_acm \
+  stdio_usb_serial_jtag \
```

Comment:
The same here?

Reply:
No, this is just a list of all possible stdio modules to select `stdio_uart` if none of them is selected.

Reply:
... required for the whether to enable `stdio_acd_acm`.